src/core/libmaven/EIC.cpp

```diff
@@ -278,6 +285,15 @@ void EIC::computeThresholdBaseline(int smoothingWindow, int dropTopX)
     //cerr << "eic no noise obs = " << eic_noNoiseObs << "\n";
 }
 
+void EIC::computeBaseline()
+{
+    if (_baselineMode == BaselineMode::Threshold)
```

Comment:
I always prefer Switch case  over if/else if/else where ever it's possible. , especially in the case of Enums

Reply:
Sure.